mosip/on-prem/README.md

```diff
@@ -190,14 +190,16 @@ Note: Before adding/removing nodes make sure that ```rke version``` should be sa
   _Note: This step is required only if you have to add nodes to an existing cluster._
   * Copy the ssh keys, setup Docker and open ports as given above.
   * Edit the `cluster.yml` file and add extra nodes with their IPs and roles.
-  * Run `rke up --update-only` to bring up the changes to the cluster.
+  * Execute the command `rke up --update-only` to add/remove nodes consisting of only `worker` role .
+  * Execute the command `rke up` to add nodes consisting of `worker`, `etcd`, `controlplane` role.
```

Comment:
in case node to be added have etcd as well as worker node . what should be used?

Also rather then differentiating can't we streamline the same to use `rke up` onlu ?